platform-configs/include/protectli-common.robot

```diff
@@ -1,5 +1,6 @@
 *** Settings ***
 Resource    default.robot
+Resource    ../../keys.robot
```

Comment:
Is this needed? I don't see any of the keys being used in the new test

Reply:
```
==============================================================================
Check-Ethernet-Ports-Order                                                    
==============================================================================
[ ERROR ] Error in file '/home/wmowinski/open-source-firmware-validation/platform-configs/include/protectli-common.robot' on line 12: Setting variable '${BOOT_MENU_KEY}' failed: Variable '${F11}' not found.
SPS001.001 Ethernet ports are in order :: This test automates the ... | FAIL |
Variable '${BOOT_MENU_KEY}' not found.
------------------------------------------------------------------------------
Check-Ethernet-Ports-Order                                            | FAIL |
1 test, 0 passed, 1 failed
```
apparently, it is

Reply:
`keys.robot` does not contain `BOOT_MENU_KEY` variable. `BOOT_MENU_KEY` should be defiend i nthe paltform config.

Reply:
`keys.robot` is only included as a resource in the tests, not platform configs currently. Please add it to `dasharo-security/sort-pci-semiauto.robot` to keep consistency with other tests.

Reply:
https://github.com/Dasharo/open-source-firmware-validation/pull/515/commits/18d1c5f8639fae1cddb4cd28aaa9e25382e1d8ff